### ISSUE TITLE
displayColour syntax not checked well and can give false error messages

### ISSUE THREAD

**Issue by TimLethbridge** [umple/Umple#2056]
Any of the following will result in a message stating:

dot (part of GraphViz) does not appear to be installed on the server

when in fact the problem is that the parsing code has picked up an invalid colour. The colour should be an html colour name and can be a hash followed by a 6-character hex code.

(displayColor or displayColour are interchangable ... British/American spelling)

Case 1: Empty colour (several ways this can be generated). Note that the equals sign is removed if it appears and the colour is whatever comes next up to the semicolon. There should be a check for an invalid colour, or else just ignore an empty string.
```
displayColor;
displayColor=;
```

Case 2: After an equals sign there is a space before quotes. Note that if quotes are not present, they are added, but. Trim all spaces after removing the equals sign,.

  displayColor= "red";

Also change the 'does not appear to be installed on the server message, so it is more meaningful ... it should say "has encounteres an error processing a gv or else may not be installed"

**Comment by AxelTWC** [comment]
# New PR above
The PR above current does not include test cases yet.
@syedahmed1674 Will Push Test Cases later and will make a comment again


## PR Title 
Support the implementation of Display Color assignment with Parenthesis.
## Example
displayColor = "red" <-- This does not work originally 

## PR Description
It was originally stated by @syedahmed1674 that by implementing the following into UmpleInternaalParser_CodeLayout + _CodeClass:
```
theColor = theColor.replace("=", ""); // Remove all '=' characters
theColor = theColor.replace(""", ""); // Remove all '"' characters
```
It would fix the issue by replacing any parsed values for the Coloring to remove the assignment sign and the quotation sign, It was later found that it does not pass through Grammar Test Cases and raises error 1502. 

It was later discovered by @AxelTWC debugging that by removing the code:
```
      if(!theColor.startsWith("\"")) theColor= "\""+theColor;
      if(!theColor.endsWith("\"")) theColor= theColor+"\"";
``` 

It solves the issue where there will not be an extra " whenever having a quotation valued assignment 
(Originally Debugging Gives " "red" by doing displayColor = "red" | It should fix the issue now)
A full build was ran and passes all previous cases in prior to making sure this works perfectly.

@syedahmed1674 Will Push Test Cases Specifically for this PR #2060 /Issue #2056

**Comment by TimLethbridge** [comment]
Fixed by #2060
